core/parentchain/light-client/src/lib.rs

```diff
@@ -25,6 +25,7 @@ compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the sam
 #[cfg(all(not(feature = "std"), feature = "sgx"))]
 #[macro_use]
 extern crate sgx_tstd as std;
+extern crate core;
```

Comment:
CLion adds this statement (unnecessarily) when you import certain crates from `core`. You can remove it again.

Reply:
ok. interesting :)